app/controllers/tags_controller.rb

```diff
@@ -26,7 +26,7 @@ def show
           @page_title = this_blog.tag_title_template.to_title(@tag, this_blog, params).strip
           @description = this_blog.tag_desc_template.to_title(@tag, this_blog, params).strip
           @keywords = this_blog.meta_keywords
-          render template_name(params[:id])
+          render "articles/index"
```

Comment:
Why is this hardcoded now?

Reply:
This update allows to use same template for tag as for articles to not duplicate templates and folders in main app.